### PR TITLE
aiac: add page

### PR DESCRIPTION
**Pull request by DevMan01**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented:** v1.0.0

### CODE REVIEW

File: pages/osx/aiac.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`aiac get dockerfile for a secured nginx`
+
+- Generate GitHub action that applies Terraform:
+
+`aiac get github action that plans and applies terraform`
+
+- Generate a port scanner in Python:
+
+`aiac get python code that scans all open ports in my network`
```
Comment (blueskyson) [inline pages/osx/aiac.md:20]:
```suggestion
`aiac get python {{code that scans all open ports in my network}}`
```

File: pages/osx/aiac.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a MongoDB query:
+
+`aiac get mongo query that aggregates all documents by created date`
```
Comment (blueskyson) [inline pages/osx/aiac.md:24]:
```suggestion
`aiac get mongo {{query that aggregates all documents by created date}}`
```

**Comment by DevMan01** [comment]
Changes made, ready for further review / approval.

File: pages/osx/aiac.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# aiac
+
+> Use OpenAI to generate IaC, configurations, utilities, queries and more.
```
Comment (kbdharun) [inline pages/osx/aiac.md:3]:
```suggestion
> Use OpenAI to generate IaC configurations, utilities, queries and more.
```
Is , required?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/osx/aiac.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/gofireflyio/aiac>.
+
+- Generate Terraform for Azure storage account:
+
+`aiac get terraform for an azure storage account`
+
+- Generate a Dockerfile for nginx:
+
+`aiac get dockerfile for a secured nginx`
```
[the remark this answers is not in the document]
Comment (blueskyson) [reply pages/osx/aiac.md:12]: This haven't been applied.

**Comment by DevMan01** [comment]
@blueskyson - We good for approval?

**Review by blueskyson — APPROVED** [review]

**Review by mfrw — COMMENTED** [review]
Ohh wow .. TIL
@DevMan01 Thanks for your contribution.

**Pull request merged**